tests-next-spark/README.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Testing
+nav_order: 1
+---
+# RAPIDS Accelerator for Testing against the upcoming version of Apache Spark
+
+While writing unit-tests, we can run into situations where we depend on classes that are not
+ available in the default version of Spark (currently 3.0.0). In such a scenario we put
```

Comment:
I'd refrain from putting in a version in here which is immediately out of date (3.0.1 is actually the "current" version).  Instead I suggest focusing on the snapshot nature of it, e.g.:
```suggestion
While writing unit-tests, we can run into situations where we depend on classes that are only
available in the SNAPSHOT version of the upcoming next version of Spark. In such a scenario we put
```